{
  "Header": {
    "Original": "CVE-2015-4603 (The exception::getTraceAsString function in Zend/zend_exceptions.c in  ...)",
    "Line": 438345,
    "ID": "CVE-2015-4603",
    "Description": "(The exception::getTraceAsString function in Zend/zend_exceptions.c in  ...)"
  },
  "Annotations": [
    {
      "Original": "- php5 5.6.9+dfsg-1",
      "Line": 438346,
      "Type": "package",
      "Package": "php5",
      "Kind": "fixed",
      "Version": "5.6.9+dfsg-1"
    },
    {
      "Original": "[jessie] - php5 5.6.9+dfsg-0+deb8u1",
      "Line": 438347,
      "Type": "package",
      "Release": "jessie",
      "Package": "php5",
      "Kind": "fixed",
      "Version": "5.6.9+dfsg-0+deb8u1"
    },
    {
      "Original": "[wheezy] - php5 5.4.41-0+deb7u1",
      "Line": 438348,
      "Type": "package",
      "Release": "wheezy",
      "Package": "php5",
      "Kind": "fixed",
      "Version": "5.4.41-0+deb7u1"
    },
    {
      "Original": "NOTE: https://bugs.php.net/bug.php?id=69152 [2015-03-03 04:30 UTC]",
      "Line": 438349,
      "Type": "NOTE",
      "Description": "https://bugs.php.net/bug.php?id=69152 [2015-03-03 04:30 UTC]"
    }
  ]
}